src/js/intlTelInput.js.ejs

```diff
@@ -761,6 +765,33 @@ Plugin.prototype = {
     // dropdownHeight - 1 for border
     var cssTop = (!dropdownFitsBelow && dropdownFitsAbove) ? "-" + (this.dropdownHeight - 1) + "px" : "";
     this.countryList.css("top", cssTop);
+    
+    // if container, append countryList to this.$drop and this.$drop to container
+    if (this.options.container) {        
+      this.$drop = $("<div>", {
+        "class": "intl-tel-input iti-container"
+      });
+      
+      var $dropInner = $("<div>", {
+        "class": "flag-dropdown"
+      }).appendTo(this.$drop),
+        inputHeight = !dropdownFitsBelow && dropdownFitsAbove ? 0 : that.telInput.outerHeight();
+      
+      // calculate placement
+      that.$drop.css({
+        "top": pos.top + inputHeight - parseInt(this.telInput.css('borderTopWidth'), 10) - parseInt(this.telInput.css('borderBottomWidth'), 10),
+        "left": pos.left,
+        "position": "absolute"
+      });
+      
+      // close menu on resize/scroll
+      this.telInput.parents().off("resize" + this.ns + " scroll" + this.ns).on("resize" + this.ns + " scroll" + this.ns, function () {
+        if (!that.countryList.hasClass("hide")) that._closeDropdown();
+      });
+      
+      this.$drop.appendTo(this.options.container);
```

Comment:
I see you inject the markup into the container in `_setDropdownPosition`, which is called every time you open the dropdown. Instead couldn't we just do this once, in `_generateMarkup`, and just show/hide it each time (and update the CSS top/left values)?